scripts/checkeol.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash 
+
+function checkEol {
+  local X=$(git --no-pager diff --name-only -M100% --diff-filter=AM --relative $(git merge-base FETCH_HEAD origin/HEAD) FETCH_HEAD | grep "\.js$")
```

Comment:
We should probably also check JSON files, and might as well check all file types, so we can get rid of the `grep`. Should use a better variable name than `X`.